README.md

```diff
@@ -1,6 +1,6 @@
-# React Native Prisma
+# Early Access: Prisma ORM for React Native and Expo
 
-A Prisma engine adaptation for React Native. Please note that this is a [Preview Feature](https://www.prisma.io/docs/orm/reference/preview-features/client-preview-features)
+A Prisma engine adaptation for React Native. Please note that this is in [Early Access](https://www.prisma.io/docs/orm/reference/preview-features/client-preview-features)
```

Comment:
link to https://www.prisma.io/docs/orm/more/releases#early-access instead of preview features?

Reply:
```suggestion
A Prisma engine adaptation for React Native. Please note that this is in [Early Access](https://www.prisma.io/docs/orm/more/releases#early-access)
```